frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Remove the default readme description and update your description, library used, api used for data, add few screenshots of the components.